packages/xo-web/src/common/intl/messages.js

```diff
@@ -114,6 +114,7 @@ const messages = {
   // ----- Filters -----
   onError: 'On error',
   successful: 'Successful',
+  filterOutSomeTasks: 'Hide some tasks',
```

Comment:
`Hide some tasks` look weird no ? What do you think of something more explicit ?

Reply:
Yes, something like "Hide short tasks".